config/docs-json.config.js

```diff
@@ -56,7 +102,7 @@ function getDocsShape() {
       name: humanCase(token.path.slice(1).map(humanCase).join(' / ')),
       cssVariableName: `--${token.name}`,
       jsPathName: token.path.map(camelCase).join('.'),
-      contrastChecks: token.extensions?.telekom?.docs?.contrast,
+      contrastChecks: token.extensions?.telekom?.docs?.contrast.map(el => getContrastCheck(token.value, el, allTokens))
```

Comment:
I would rename `el` to `path`

Reply:
for sure, i will add some more meaningful variable names with the next commit